modules/cactus-web/src/Notification/Notification.story.tsx

```diff
@@ -0,0 +1,86 @@
+import { select } from '@storybook/addon-knobs'
+import { Meta } from '@storybook/react/types-6-0'
+import { Page } from 'puppeteer'
+import React from 'react'
+
+import Alert from '../Alert/Alert'
+import Button from '../Button/Button'
+import Notification from './Notification'
+
+export default {
+  title: 'Notification',
+  component: Notification,
+} as Meta
+
+export const BasicUsage = (): React.ReactElement => {
+  const [open, setOpen] = React.useState<boolean>(false)
+
+  const handleOpen = () => setOpen(true)
+  const handleClose = () => setOpen(false)
+
+  const vertical: 'top' | 'bottom' = select('vertical', ['top', 'bottom'], 'bottom')
+  const horizontal: 'left' | 'center' | 'right' = select(
+    'horizontal',
+    ['left', 'center', 'right'],
+    'right'
+  )
+
+  return (
+    <>
+      <Button id="open-notification" onClick={handleOpen}>
+        Open Notification
+      </Button>
+      <Notification open={open} position={{ vertical, horizontal }}>
+        <Alert status="error" onClose={handleClose}>
+          Message
+        </Alert>
+      </Notification>
+    </>
+  )
+}
+
+BasicUsage.parameters = {
+  beforeScreenshot: async (page: Page) => {
+    await page.click('button[id="open-notification"]')
```

Comment:
I don't know if the ID selector is actually more efficient, but 🤷.
```suggestion
    await page.click('#open-notification')
```